website/content/docs/secrets/ad/migration-guide.mdx

```diff
@@ -109,6 +109,14 @@ the chance of this happening. Additionally, tuning the AD secrets engine [last_r
 parameter could help mitigate applications reading stale passwords, since the parameter allows
 rotation of the password if it's been rotated out-of-band within a given duration.
 
+##### Lazy vs Auto password rotation
```

Comment:
```suggestion
```

Suggest removing the heading since the entire section is a note, which will make the information stand out anyway